src/main/scala/db/migration/V5__RemoveIngressVisPaaSiden.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Part of NDLA article_api.
+ * Copyright (C) 2016 NDLA
+ *
+ * See LICENSE
+ *
+ */
+
+package db.migration
+
+import java.sql.Connection
+
+import org.flywaydb.core.api.migration.jdbc.JdbcMigration
+import org.json4s.JsonAST.{JArray, JField, JObject}
+import org.json4s.native.JsonMethods._
+import org.json4s.native.Serialization._
+import org.postgresql.util.PGobject
+import scalikejdbc._
+
+class V5__RemoveIngressVisPaaSiden extends JdbcMigration {
```

Comment:
Det er flyway sin default navnekonvensjon for migration scripts: https://flywaydb.org/documentation/migration/sql . Navnekonvensjonen er konfigurerbar, men vi bruker altså den som er default:)